tomcat/src/main/java/com/techcourse/controller/LoginController.java

```diff
@@ -0,0 +1,49 @@
+package com.techcourse.controller;
+
+import com.techcourse.db.InMemoryUserRepository;
+import com.techcourse.model.User;
+import org.apache.coyote.controller.AbstractController;
+import org.apache.coyote.file.ResourcesReader;
+import org.apache.coyote.http11.cookie.Cookies;
+import org.apache.coyote.http11.path.Path;
+import org.apache.coyote.http11.request.HttpRequest;
+import org.apache.coyote.http11.response.HttpResponse;
+import org.apache.coyote.http11.session.Session;
+
+public class LoginController extends AbstractController {
+    @Override
+    protected void doGet(final HttpRequest request, final HttpResponse response) {
+        response.setResource(ResourcesReader.read(Path.from("login.html")));
```

Comment:
```suggestion
        response.setResource("login.html");
```

클라이언트가 어떤 객체를 넘겨줘야할지 판단할 필요 없도록 만들면 어떨까요